### PR TITLE
Genomic manual-sample tool

### PR DESCRIPTION
**Pull request by j-kanuch**
This PR includes a quick tool for inserting new genomic set member records to support the E2E test. This tool can be expanded upon for more control in future PRs.

### CODE REVIEW

File: rdr_service/tools/tool_libs/genomic_utils.py [hunk 1]
```diff
@@ -493,6 +493,86 @@ def run(self):
...
+    def run(self):
+        """
+        Main program process
+        :return: Exit code value
+        """
+
+        # Validate Aruguments
+        if self.args.csv is None:
+            _logger.error('Argument --csv must be provided.')
+            return 1
```
Comment (kenny-skaggs) [inline rdr_service/tools/tool_libs/genomic_utils.py]: Could this be done by requiring the argument on the sub-parser?

File: rdr_service/tools/tool_libs/genomic_utils.py [hunk 2]
```diff
@@ -493,6 +493,86 @@ def run(self):
...
+                    session.flush()
+
+                    for line in lines:
+                        if len(line.split(',')) > 0:
+                            _pid = line.split(',')[0].strip()
+                            _bid = line.split(',')[1].strip()
+                            _sample_id = line.split(',')[2].strip()
+                            _sab = line.split(',')[3].strip()
+                            _siteId = line.split(',')[4].strip()
+                            _state = line.split(',')[5].strip()
```
Comment (kenny-skaggs) [inline rdr_service/tools/tool_libs/genomic_utils.py]: Would a csvreader work for parsing the file?

**Review by kenny-skaggs — COMMENTED** [review]
Looks good, I just had a couple of coding style type of comments

File: rdr_service/tools/tool_libs/genomic_utils.py (hunk 2 above)
Comment (kenny-skaggs) [reply rdr_service/tools/tool_libs/genomic_utils.py]: I thought that was the name of some class, but it looks like it doesn't really exist. `csv.reader` https://docs.python.org/3/library/csv.html is what I meant by it.

Comment (j-kanuch) [reply rdr_service/tools/tool_libs/genomic_utils.py]: Done.

File: rdr_service/tools/tool_libs/genomic_utils.py (hunk 1 above)
Comment (j-kanuch) [reply rdr_service/tools/tool_libs/genomic_utils.py]: Done.

**Review by kenny-skaggs — APPROVED** [review]

**Pull request merged**